src/mpc/structs.jl

```diff
@@ -221,27 +252,30 @@ end
 Base.@kwdef struct MPCElectricStorage < AbstractElectricStorage
     size_kw::Float64
     size_kwh::Float64
-    charge_efficiency::Float64 =  0.96 * 0.975^2
-    discharge_efficiency::Float64 =  0.96 * 0.975^2
+    charge_efficiency::Float64 =  0.96 * 0.975^0.5
```

Comment:
why don't we have internal, rectifier, and inverter efficiencies like reg ElectricStorage?